src/Axe/wwwroot/js/compiler.observe.js

```diff
@@ -0,0 +1,154 @@
+﻿// Declarations
+var messageTypes = {
```

Comment:
variables and functions in this script become member of a global object. encapsulate them in a function instead and assign handler to document event `$(document).ready(function() { /*current code*/ })`